src/asciidoc/docs/platform-features/richPreview.adoc

```diff
@@ -1,143 +1,61 @@
 = Thumbnail image and custom metadata
 :page-slug: thumbnail-image
-:page-description: How to customize the thumbnail (Open Graph) image and sharing metadata for your Koji app.
+:page-description: How to customize the thumbnail (Open Graph) image and metadata for your Koji app.
 :page-banner: {imagesDir}/Koji-screenshot=1.png
 
-When a user shares a remix of your template on social media, the Koji platform renders a rich preview (Open Graph) image based on a screenshot of the Koji.
-By default, the platform takes a screenshot of the initial view.
+When you share a Koji app on social media, the Koji platform renders a thumbnail (Open Graph) image based on a screenshot of the app.
+If you were designing a web app from scratch, you would use meta tags, such as `og:image`, `og:title`, and `og:description`, to set properties for the thumbnail.
+The Koji platform automatically sets these properties for you, but you can override some of Koji's default settings to provide a customized thumbnail.
 
-In some cases, the initial view might not be the best preview image for your template.
-For example, if a game template starts with an instruction screen, a screenshot of the first level might provide a better preview image.
-To improve the sharing experience in these cases, you can render a custom view of your template that is designed specifically to make an impression when sharing a remix.
+By default, the platform takes a screenshot of the initial view.
+In some cases, the initial view might not be the best preview image for your app.
+For example, if a game app starts with an instruction screen, a screenshot of the first level might provide a better preview image.
+To improve the sharing experience in these cases, you can render a custom view of your app that is designed specifically to make an impression when sharing it.
 You can also control the timing of the screenshot to ensure that all relevant assets have loaded before the preview image is generated.
 
-You can also customize the sharing metadata by configuring an entitlement.
+You can also give creators the ability to customize the thumbnail by enabling <<entitlements#_custom_metadata,custom metadata>>.
```

Comment:
If we're using only a link and not providing any information beyond what is in the Entitlements page, it's probably better to make this page focus only on customizing the thumbnail image, rather than covering all the sharing metadata. If we go that route, specific changes would be:

- Change the title (and maybe the title of the first heading, so they are not duplicates). Something like "Customizing the thumbnail image" for the page and something like "Rendering a custom view for the screenshot" as the first heading
- Remove the last section on the page (currently called "Customizing the thumbnail metadata")
- Make this last sentence of the intro into a tip, since it's not directly related to the page anymore but still interesting to someone who wants to improve the sharing experience.